src/git-town

```diff
@@ -33,4 +33,5 @@ elif [ -n "$1" ]; then
   echo_inline_usage_or "git town main-branch [<branchname>]"
   echo_inline_usage_or "git town non-feature-branches [(--add | --remove) <branchname>]"
   echo_inline_usage_or "git town version"
+  exit_with_error
```

Comment:
This adds an extra empty line that I don't think should be there. 
`exit_with_error` is misnamed but thats what that function does right now.